core/front/models/class-model-header.php

```diff
@@ -165,18 +165,36 @@ function czr_fn_extend_params( $model = array() ) {
     * 3) display in desktop and not in mobile
     * 4) display in mobile and desktop
     */
+    /*
+      Imp: update model and templates to the latest b4 stable
+      in the latest b4 stable the hidden-{breakpoint}-(down|up)
+      have been removed.
+      This, I think, because of the "half pixel" issue with media-queries.
+      Now the logic, always mobile first, can be summed up in the following
+      examples:
+      - if you want to hide something for smartphones, but display it elsewhere
+      (sm up), apply the following classes to your element:
+      a) d-none
+      b) d-lg-(flex|block|inline-block...)*
+      *meaning that you need to exactly know how to display it in lg, either flex
+      or block, etc.
+
+      - say you want to display something in mobiles (tablets and spartphones)
+      but not on large devices (desktop) you just need to apply this class to your element
+      d-lg-none
+    */
```

Comment:
ok thanks for the very clear explanations.
Good improvement in B4 I think

Reply:
yes, not really noticeable if not in edge cases, but I think is a technically good improvement.
Basically they did what we discussed, and somewhat planned to do, when we talked about the "half pixel" issue about one year ago. It's good to see that we ended up with the same solution (even if we didn't implemented at that time :D).